client/lib/plans/features-list.js

```diff
@@ -801,52 +801,52 @@ export const FEATURES_LIST = {
 
 	[ constants.FEATURE_CORE_JETPACK ]: {
 		getSlug: () => constants.FEATURE_CORE_JETPACK,
-		getTitle: () => i18n.translate( 'Core Jetpack Services' ),
+		getTitle: () => i18n.translate( 'Core Jetpack services' ),
```

Comment:
:ok: This change will be queued for retranslation. We'll use the existing translations in the meantime.